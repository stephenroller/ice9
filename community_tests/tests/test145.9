# proc overriding global scope w/ param					succeed

var bar : int

proc foo(bar : bool)
	bar := true;
	write ?bar;
end

bar := 0;

foo(true);

write bar;
