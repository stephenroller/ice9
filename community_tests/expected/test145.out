Loading file: generated/test145.tm
1 
0 
Number of instructions executed = 49
